stdlib/REPL/src/TerminalMenus/AbstractMenu.jl

```diff
@@ -362,10 +368,15 @@ down_arrow(c::AbstractConfig) = down_arrow(c.config)
 down_arrow(c::Config) = c.down_arrow
 down_arrow(::AbstractMenu) = CONFIG[:down_arrow]
 
-print_arrow(buf, ::ConfiguredMenu, c::Char) = print(buf, c, "  ")
+updown_arrow(m::ConfiguredMenu) = updown_arrow(m.config)
+updown_arrow(m::AbstractConfig) = updown_arrow(c.config)
+updown_arrow(c::Config) = c.updown_arrow
+updown_arrow(::AbstractMenu) = CONFIG[:updown_arrow]
+
+print_arrow(buf, ::ConfiguredMenu, c::Char) = print(buf, c)
```

Comment:
```suggestion
```
Do we even need `print_arrow` anymore?

Reply:
Good point! They're now replaced by plain `print`.